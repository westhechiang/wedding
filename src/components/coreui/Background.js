import styled from 'styled-components';
import { theme } from '../../ui';

const Background = styled.div`
  width: 100%;
  height: ${props => props.height || '100%'};
  background: url(${props => props.content}) ${theme.color.black} no-repeat center center fixed;
  background-size: cover;
`;

export default Background;
