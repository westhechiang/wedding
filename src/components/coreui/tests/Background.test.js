import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Background from '../../../components/coreui/Background';

describe('<Background />', () => {
  it('should match its empty snapshot', () => {
    const wrapper = shallow(<Background />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
