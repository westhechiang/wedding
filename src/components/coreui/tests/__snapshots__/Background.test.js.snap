// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Background /> should match its empty snapshot 1`] = `
<div
  className="sc-bdVaJa dqUYqv"
/>
`;
